core/src/main/java/org/apache/carbondata/core/constants/CarbonLoadOptionConstants.java

```diff
@@ -62,7 +62,7 @@
   public static final String CARBON_OPTIONS_DATEFORMAT =
       "carbon.options.date.format";
 
-  public static final String CARBON_OPTIONS_DATEFORMAT_DEFAULT = "";
+  public static final String CARBON_OPTIONS_DATEFORMAT_DEFAULT = "yyyy-MM-dd";
```

Comment:
It's better use CarbonCommonConstants.CARBON_DATE_DEFAULT_FORMAT as default value.